templates/agenda/protocol.html.twig

```diff
@@ -0,0 +1,35 @@
+{% extends 'layout-with-agenda-submenu.html.twig' %}
+
+{% trans_default_domain 'agenda' %}
+
+{% block title %}{% trans %}Protocol{% endtrans %}{% endblock %}
+
+{% block header %}
+    <div class="row">
+        <div class="col">
+            <h1>{% trans %}Protocol{% endtrans %}</h1>
+            <p class="text-muted"><small>{% trans %}Create or update protocol{% endtrans %}</small></p>
+        </div>
+        <div class="col-auto">
+            {# TODO: Needs functionality on buttons #}
+{#            <a name="" id="" class="btn btn-success" href="#" role="button">{% trans %}Create inspection letter{% endtrans %}</a>#}
+            {#            <a name="" id="" class="btn btn-primary" href="#" role="button">{% trans %}Save changes{% endtrans %}</a>#}
```

Comment:
Do we still need this comment?

Reply:
Removed redundant commented code and `TODO` .